HACKING.md

```diff
@@ -84,16 +85,16 @@ switch the Core Selector to “`release`” and click on “Query” to get to
 <http://localhost:8983/solr/#/release/query>.
 
 Then start filling the form as follows:
-* Set the first field “Request-Handler (qt)” (qt)
+* Set the first field `Request-Handler (qt)`
```

Comment:
It’s full text in the query screen, not code, so the English double quotation marks seem more appropriate.

Reply:
See https://solr.apache.org/guide/7_7/images/query-screen/query-top.png